su-template/default/layout/html/headers.erb

```diff
@@ -0,0 +1,15 @@
+  <meta charset="<%= charset %>">
+  <!-- VIEWPORT -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
This meta tag - what's its purpose? Would this affect the responsiveness for the pages?